configure.ac

```diff
@@ -43,6 +43,12 @@ AC_GNU_SOURCE
 
 CFLAGS="$CFLAGS -std=c99 -Wall -pedantic -Wextra -Werror -pedantic-errors"
 
+# pledge (OpenBSD)
```

Comment:
I think we should just say `# pledge` here (in case it's ported elsewhere -- I know there is some sort of port of the API to Linux, so maybe it'll become wider spread over time).

Reply:
Done.